### PR TITLE
fix: Sass has not been compiled when tests running

### PR DESCRIPTION
**Pull request by ValeraS**
have made: Sass compiling before tests run #216

### CODE REVIEW

File: src/templates/Challenges/rechallenge/transformers.js [hunk 1]
```diff
@@ -107,12 +88,34 @@ export const babelTransformer = cond([
...
-  [testSASS, partial(vinyl.appendToTail, browserSassCompiler)],
+const htmlSassTransformCode = file => {
+  let doc = document.implementation.createHTMLDocument();
+  doc.body.innerHTML = file.contents;
+  let styleTags = [].filter.call(
+      doc.querySelectorAll('style'),
+      style => style.type === 'text/sass'
+  );
+  if (styleTags.length === 0 || typeof Sass === 'undefined') {
+    return vinyl.transformContents(() => doc.body.innerHTML, file);
```
Comment (Bouncey) [inline src/templates/Challenges/rechallenge/transformers.js]: Why update the file is we are not changing anything?

File: src/templates/Challenges/rechallenge/transformers.js [hunk 2]
```diff
@@ -107,12 +88,34 @@ export const babelTransformer = cond([
...
+};
+
+export const htmlSassTransformer = cond([
+  [testHTML, htmlSassTransformCode],
   [stubTrue, identity]
 ]);
 
-export const _transformers = [replaceNBSP, babelTransformer, sassTransformer];
+export const _transformers = [
+  replaceNBSP, babelTransformer, htmlSassTransformer];
```
Comment (Bouncey) [inline src/templates/Challenges/rechallenge/transformers.js]: Formatting here.

To clear the `max-len` eslint rule, it is more readable to do

```js
export const _transformers = [
  replaceNBSP,
  babelTransformer,
  htmlSassTransformer
];
```

File: src/templates/Challenges/rechallenge/transformers.js (hunk 1 above)
Comment (ValeraS) [reply src/templates/Challenges/rechallenge/transformers.js]: it's a fix for an unintended code shown https://github.com/freeCodeCamp/learn/pull/218

File: src/templates/Challenges/rechallenge/transformers.js (hunk 2 above)
Comment (ValeraS) [reply src/templates/Challenges/rechallenge/transformers.js]: should I amend or add a new commit?

Comment (nikrb) [reply src/templates/Challenges/rechallenge/transformers.js]: by all means wait for the word of the Bouncey one, but I'd say add a new commit. No expert but amending commits that have been pushed can be a pain.

**Comment by Bouncey** [comment]
@ValeraS I have tried to fix conflicts in the github UI but it didn't like it. Could you please do a rebase on this? 

Many thanks 👍

**Comment by ValeraS** [comment]
@Bouncey It's done.

**Comment by Bouncey** [comment]
Awesome work @ValeraS Thank you 👍 

I sent you a message on gitter, just wanted to be sure you have seen it.

**Pull request merged**